pages/Vans/Vans.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
```

Comment:
It seems like you are no longer using the useState and useEffect hooks from the react-router in this file. You can remove that  import statement.